packages/sources/README.md

```diff
@@ -10,12 +10,15 @@ This document was generated automatically. Please see [Master List Generator](..
 - [alongside](./alongside/README.md)
 - [alphachain](./alphachain/README.md)
 - [alphavantage](./alphavantage/README.md)
+- [alphavantage-test](./alphavantage-test/README.md)
 - [alpine](./alpine/README.md)
 - [amberdata](./amberdata/README.md)
 - [anyblock](./anyblock/README.md)
 - [ap-election](./ap-election/README.md)
+- [armanino](./armanino/README.md)
```

Comment:
It seems that you had some old local folders for deleted EAs. 
`git reset --hard origin/main`  should sync the local state with origin, after that you can re-run masterlist and readme generation scripts

Reply:
Good catch. . Fixed now 🙏